flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/runtime/stream/sql/RankITCase.scala

```diff
@@ -1095,34 +1070,35 @@ class RankITCase(mode: StateBackendMode) extends StreamingWithStateTestBase(mode
     assertEquals(updatedExpected.sorted, sink.getUpsertResults.sorted)
   }
 
-  // FIXME
-  @Ignore("Enable after agg implements ExecNode")
   @Test
   def testMultipleRetractTopNAfterAgg(): Unit = {
```

Comment:
I would like to ignore this test and revert the changes, because current implementation lost the original test purpose, i.e. "test multiple retract sink after topN".